playwright-e2e/pages/add-and-remove-admin-case-flag.ts

```diff
@@ -0,0 +1,49 @@
+import { expect, type Locator, type Page } from "@playwright/test";
+import { BasePage } from "./base-page";
+import config from "../settings/test-docs/config";
+
+export class AddAndRemoveAdminCaseFlag extends BasePage {
+
+    readonly uploadAssessmentForm: Locator;
+    readonly additionalNotes: Locator;
+    readonly addOrRemoveFlag: Locator;
+    readonly yes: Locator;
+    readonly no: Locator;
+
+    public constructor(page: Page) {
+        super(page);
+        this.yes =  page.getByLabel('Yes');
+        this.uploadAssessmentForm = page.getByRole('textbox', { name: 'Upload assessment form or' });
+        this.additionalNotes =  page.getByLabel('Additional notes (Optional)');
+        this.addOrRemoveFlag =  page.getByRole('link', { name: 'Add or remove case flag' });
+        this.no =  page.getByLabel('No', { exact: true });
+    }
+
+    public async addCaseFlag() {
+        await this.yes.check();
+        await this.uploadAssessmentForm.setInputFiles(config.testPdfFile);
```

Comment:
do we need to have the wait for upload to be finished code after this?

Reply:
added wait